templates/apollo-starter-ts/docker-compose.yml

```diff
@@ -12,8 +12,8 @@ services:
 
   # Mosca is a simple MQTT Broker
   # In OpenShift/Production we would use the Red Hat AMQ broker
-  mosca:
-    image: matteocollina/mosca
-    ports:
-      - "1883:1883" # MQTT
-      - "80:80" # web interface
+  # mosca:
```

Comment:
We will need that for subscriptions. @matteocollina/mosca is the best MQTT broker for this job. Lets leave it